client/controllers/WebPublisherOutputController.js

```diff
@@ -71,11 +71,14 @@ export function WebPublisherOutputController($scope, $sce, modal, publisher, aut
         websocketOpen() {
             let pubConfig = config.publisher || {};
 
-            let domain = pubConfig.wsDomain ? pubConfig.wsDomain : `${pubConfig.tenant}.${pubConfig.domain}`;
+            let protocol = (pubConfig.wsProtocol && pubConfig.wsProtocol.length > 0) ? `${pubConfig.wsProtocol}` : 'wss';
+            let subdomain = (pubConfig.tenant && pubConfig.tenant.length > 0) ? `${pubConfig.tenant}.` : '';
```

Comment:
tomaszrondio,

If tenant info is stuffed into wsDomain then why not wsProtocol? And then why not wsPort?
ok to diverge from HTTP precedent already there?

Personally, I prefer consistency with HTTP precedent. I leave it to you to decide.

Reply:
we need to keep websocket configuration separate as with many tenants it can have totally different url than default tenant configuration. Only that, rest of Your PR is good and we would merge it.

Reply:
ok, will send out a new pull request.